include/dsn/dist/failure_detector/failure_detector.h

```diff
@@ -105,7 +105,7 @@ class failure_detector :
 
     void register_master(::dsn::rpc_address target);
 
-    bool switch_master(::dsn::rpc_address from, ::dsn::rpc_address to);
+    bool switch_master(::dsn::rpc_address from, ::dsn::rpc_address to, uint32_t delay_milliseconds);
```

Comment:
I don't get it. Normally, if the master is incorrect, we'd love to connect with the new master asap.